models/src/anemoi/models/layers/diffusion.py

```diff
@@ -0,0 +1,38 @@
+# (C) Copyright 2024 Anemoi contributors.
```

Comment:
new files should use the updated header for this year ``Copyright 2025 Anemoi contributors.``